src/docs/Components/Accordion/code.mdx

```diff
@@ -112,4 +112,79 @@ Then just add the CSS class `mc-accordion__icon` to your icon.
 </div>
 ```
 
-<Preview path="icon" />
\ No newline at end of file
+<Preview path="icon" />
+
+### With Toogle
+
+You can add a Mozaic toggle into any accordions you want.
+
+For this you have to add to the label the `mc-accordion__header` modifier.
+
+Then just add the CSS class `mc-accordion__selectable` to your toogle.
+
+```html
+<div class="mc-accordion">
+  <input type="checkbox" id="toggle" class="mc-accordion__trigger" />
+  <label
+    class="
+      mc-accordion__label mc-accordion__header
+    "
+    for="toggle"
+  >
+    <div class="mc-toggle mc-accordion__selectable">
+      <input class="mc-toggle__input" id="toggle" type="checkbox" />
+      <label class="mc-toggle__label" for="toggle"> </label>
+    </div>
+    <span aria-expanded="false">Accordion with toggle </span>
+  </label>
+  <div class="mc-accordion__content">
+    <p>
+      Fusce iaculis dolor nulla, a maximus ipsum sollicitudin et. Ut
+      condimentum at orci aliquam feugiat. Curabitur sagittis placerat leo sit
+      amet pharetra.
+    </p>
+  </div>
+</div>
+```
+
+<Preview path="selectable" />
```

Comment:
@mohamedMok I have a problem with the toggle. When I clicked on it, it opened the accordion instead of switch to the checked state.